src/test/java/com/google/api/codegen/testsrc/library_gapic.yaml

```diff
@@ -248,9 +248,13 @@ interfaces:
       - shelf.name=Math
       - series_uuid.series_string=xyz3141592654
       output_specs:
-      - print:
-        - "book names %s"
-        - $resp.book_names
+      - loop:
+          variable: book_name
```

Comment:
itty-nitty suggestion: we could do `title` here for concision and readability, but no biggie either way.